rskj-core/src/main/java/org/ethereum/core/Transaction.java

```diff
@@ -615,4 +615,13 @@ public static Transaction create(byte[] nonce, Web3.CallArguments args){
 
         return new Transaction(nonce, hexArgs.getGasPrice(), hexArgs.getGasLimit(), hexArgs.getToAddress(), hexArgs.getValue(), hexArgs.getData());
     }
+
+    public static RskAddress parseRskAddress(@Nullable byte[] bytes) {
```

Comment:
doesn't make sense to make it part of the `RskAddress` constructor?

Reply:
I thought about it, but I want to make sure that anyone using this constructor is very conscious about what data they're passing. This is just a wrapper around **a valid address**. The `Transaction` class has the responsibility of sanitizing the input here, with the business rules it has defined.